.github/workflows/cypress.yml

```diff
@@ -0,0 +1,16 @@
+name: cypress-test-run
```

Comment:
Can we rename the file, the `name` field, etc. to just "Test"? Like this https://github.com/RedHatInsights/ocp-advisor-frontend/blob/master/.github/workflows/test.yml

Reply:
Agree, updated the PR